lib/dai-plugin-mcd/src/QueryApiMcdService.js

```diff
@@ -0,0 +1,99 @@
+import { PublicService } from '@makerdao/services-core';
+import { ServiceRoles } from './constants';
+import { TESTNET_ID } from '../../../contracts/networks';
+import { getQueryResponse } from '../../../src/QueryApi';
```

Comment:
@levity Importing TESTNET_ID and getQueryResponse from dai.js doesn't work when doing it from the built version of the plugin.  How do you think this should be resolved?

Reply:
This should work when the plugin is built into its own module:

```js
import { TESTNET_ID } from '@makerdao/dai/contracts/networks';
import { getQueryResponse } from '@makerdao/dai/src/QueryApi';
```

To get this to work when running MCD plugin tests from within the dai.js folder, before building, though, we'll have to mess with `lib/dai-plugin-mcd/.babelrc`.

If you look at that file you'll see that it sets up the `module-resolver` plugin set up to re-route `import '@makerdao/dai'` to the correct local directory, but I don't think it will handle the more specific paths we're using above. See if you can add lines the `alias` section to get it to work, e.g.:

```json
{
  "plugins": [
    [
      "module-resolver",
      {
        "alias": {
          "@makerdao/dai": "./src",
          "@makerdao/dai/src": "./src",
          "@makerdao/dai/contracts": "./contracts",
        }
      }
    ]
  ]
}
```

Reply:
Sorry for overlooking this yesterday, btw, really should have responded to you earlier.

Reply:
Hm I'm confused.  Doing that didn't seem to change anything, the built version of QueryApiMcd still says `var _networks = require("../../../contracts/networks");` when it should say `var _networks = require("@makerdao/dai/contracts/networks");` right?

Reply:
Ah. I see now. This is the first time we're depending on `@makerdao/dai` from within a file that ends up in the build, as opposed to a test file. So `module-resolver` was always set up to do this to the build, we just never ran into it before.

So maybe there's a cleaner way to do this. But the quick-&-dirty hack that occurs to me now is to edit `scripts/prepublish.sh` like this:

```diff
+ mv ./.babelrc ./babelrc.disabled
  ./node_modules/.bin/babel -d lib/dai-plugin-mcd/dist lib/dai-plugin-mcd/src
+ mv ./.babelrc.disabled ./.babelrc
```

Reply:
If you find a better way to disable that babel plugin, or perhaps configure it so it's disabled automatically based on environment, by all means, please do. 😅

Reply:
Ok cool, yeah I misunderstood the issue initially.  Looks like we can just add a `--no-babelrc` flag to the prepublish script 😄

Reply:
Nice, that’s way better.